### PR TITLE
[RHACS] Added link to ACS support policy page

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-13822

Added link to support policy page where we mention OCP versions in the docs.

Preview:
- OS versions in the following sections:
   - https://60483--docspreview.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_ocp/prerequisites-cloud-ocp.html#acs-general-requirements_prerequisites-cloud-ocp
   - https://60483--docspreview.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_other/prerequisites-cloud-other.html#acs-general-requirements_prerequisites-cloud-other
   - https://60483--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/prerequisites-ocp.html#acs-general-requirements_prerequisites-ocp
   - https://60483--docspreview.netlify.app/openshift-acs/latest/installing/installing_other/prerequisites-other.html#acs-general-requirements_prerequisites-other
- Table at https://60483--docspreview.netlify.app/openshift-acs/latest/installing/acs-installation-platforms.html#install-platforms-methods_acs-installation-platforms

Cherrypick into:
- `rhacs-docs-4.0`
- `rhacs-docs-4.1`

### CODE REVIEW

File: modules/install-acs-operator-cloud.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ Using the OperatorHub provided with {ocp} is the easiest way to install the {pro
 
 .Prerequisites
 * You have access to an {ocp} cluster using an account with Operator installation permissions.
-* You must be using {ocp} 4.10 or later.
+* You must be using {ocp} 4.10 or later. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
Comment (msugakov) [inline modules/install-acs-operator-cloud.adoc:13]: We will be required to bump `4.10` again when the version support matrix changes. I suggest introducing a variable like `{minimum-ocp-version}` or better phrasing it in a way we don't need to revisit the version number at all.
Something like
```
You must be using {ocp} of supported version. See link for ACS version support policy.
```

File: operating/manage-vulnerabilities/scan-rhcos-node-host.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ First, {product-title-short} analyzes and detects RHCOS components. Then it matc
 [NOTE]
 ====
 * If you installed {product-title-short} by using the `roxctl` CLI, you must manually enable the RHCOS node scanning features. When you use Helm or Operator installation methods on {ocp}, this feature is enabled by default.
-* Red Hat supports scanning for RHCOS nodes only if you are using {ocp} 4.10 and later.
+* Red Hat supports scanning for RHCOS nodes only if you are using {ocp} 4.10 and later. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
Comment (msugakov) [inline operating/manage-vulnerabilities/scan-rhcos-node-host.adoc:20]: 1. Not "Red Hat supports" but "RHACS supports", right?
2. Does it make sense to have the same item twice on the page?
https://60483--docspreview.netlify.app/openshift-acs/latest/operating/manage-vulnerabilities/scan-rhcos-node-host.html

image

File: modules/rhcos-enable-node-scan.adoc [hunk 3]
```diff
@@ -9,7 +9,7 @@
 If you use {ocp}, you can enable scanning of Red Hat Enterprise Linux CoreOS (RHCOS) nodes for vulnerabilities by using {rh-rhacs-first}.
 
 .Prerequisites
-* You must use {ocp} 4.10 or later for scanning RHCOS node hosts.
+* You must use {ocp} 4.10 or later for scanning RHCOS node hosts. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
Comment (msugakov) [inline modules/rhcos-enable-node-scan.adoc:12]: Having fixed `4.10` here is probably ok, but the problem is that this item isn't specific enough. It must be a **secured cluster** of OCP 4.10+ for RHCOS node scanning to work **for that cluster**.
It does not matter where Central is running and it does not matter whether there are other non-OCP secured clusters.

File: installing/acs-installation-platforms.adoc [hunk 4]
```diff
@@ -71,7 +71,7 @@ Not all installation methods are supported for all platforms.
```
Comment (msugakov) [inline installing/acs-installation-platforms.adoc:71]: (not about this line)
I see `:ocp-supported-version: 4.5` in `modules/common-attributes.adoc`.
https://github.com/openshift/openshift-docs/blob/7033db690dbf62b5a80707441a1c40e82b69e497/modules/common-attributes.adoc#L57-L57

Maybe it's unused but it should be cleaned up or updated.

**Review by msugakov — CHANGES_REQUESTED** [review]

File: modules/acs-requirements.adoc [hunk 5]
```diff
@@ -25,10 +25,10 @@ For example, Intel processors older than _Sandy Bridge_ and AMD processors older
 To install {product-title}, you must have:
 
 ifdef::openshift[]
-* {ocp} version {ocp-supported-version} or later
+* {ocp} version {ocp-supported-version} or later. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
 endif::openshift[]
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply modules/acs-requirements.adoc:29]: Thank you Misha. There was a mismatch between context defined in the doc I'll update it to the correct context to show OCP specific prerequisites `ifeval::["{context}" == "prerequisites-ocp"]`

File: installing/acs-installation-platforms.adoc (hunk 4 above)
Comment (gaurav-nelson) [reply installing/acs-installation-platforms.adoc]: Thank you Misha. I'll create a separate PR for fixing the supported version number.

Comment (msugakov) [reply installing/acs-installation-platforms.adoc]: I don't think it is optional any more.
With the latest changes I see this https://60483--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/prerequisites-ocp.html

image

File: modules/acs-requirements.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (msugakov) [reply modules/acs-requirements.adoc]: Confirming resolved.

File: modules/acs-requirements.adoc [hunk 6]
```diff
@@ -25,10 +25,10 @@ For example, Intel processors older than _Sandy Bridge_ and AMD processors older
...
 * Cluster nodes with a supported operating system.
-For more information, see the link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
+For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
[the remark this answers is not in the document]
Comment (msugakov) [reply modules/acs-requirements.adoc]: Confirming resolved. Although, there is another problem with operating systems to be fixed.

File: modules/acs-requirements.adoc [hunk 7]
```diff
@@ -25,10 +29,14 @@ For example, Intel processors older than _Sandy Bridge_ and AMD processors older
...
 endif::openshift[]
-* Cluster nodes with a supported operating system.
-For more information, see the link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
+
+ifdef::other-platforms[]
+* A supported managed Kubernetes platform. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
+endif::other-platforms[]
+
+* Cluster nodes with a supported operating system:
 ** *Operating system*: Amazon Linux, CentOS, Container-Optimized OS from Google, {op-system-first}, Debian, {op-system-base-full}, or Ubuntu.
```
Comment (msugakov) [inline modules/acs-requirements.adoc:40]: The problem with this operating systems list is that it shows up both for OCP secured cluster installations
1. https://60483--docspreview.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_ocp/prerequisites-cloud-ocp.html
2. https://60483--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/prerequisites-ocp.html

and non-ocp secured cluster installations
3. https://60483--docspreview.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_other/prerequisites-cloud-other.html
4. https://60483--docspreview.netlify.app/openshift-acs/latest/installing/installing_other/prerequisites-other.html

The problem is that OCP itself is not supported on anything other than RHCOS and RHEL.

File: modules/acs-requirements.adoc [hunk 8]
```diff
@@ -25,10 +25,10 @@ For example, Intel processors older than _Sandy Bridge_ and AMD processors older
...
 ifdef::openshift[]
-* {ocp} version {ocp-supported-version} or later
+* {ocp} version {ocp-supported-version} or later. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
 endif::openshift[]
 * Cluster nodes with a supported operating system.
-For more information, see the link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
+For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
 ** *Operating system*: Amazon Linux, CentOS, Container-Optimized OS from Google, {op-system-first}, Debian, {op-system-base-full}, or Ubuntu.
 ** *Processor and memory*: 2 CPU cores and at least 3GiB of RAM.
 +
```
[the remark this answers is not in the document]
Comment (msugakov) [reply modules/acs-requirements.adoc:42]: Confirming resolved.

File: installing/acs-installation-platforms.adoc [hunk 9]
```diff
@@ -71,7 +71,7 @@ Not all installation methods are supported for all platforms.
 
 [.small]
 --
-1. The availability of support for each platform depends on the overarching lifecycle of the platform and the end-of-life date.
+1. The availability of support for each platform depends on the overarching lifecycle of the platform and the end-of-life date. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
[the remark this answers is not in the document]
Comment (msugakov) [reply installing/acs-installation-platforms.adoc]: Confirming resolved.

File: modules/install-acs-operator.adoc [hunk 10]
```diff
@@ -10,7 +10,7 @@ Using the OperatorHub provided with {ocp} is the easiest way to install {product
 
 .Prerequisites
 * You have access to an {ocp} cluster using an account with Operator installation permissions.
-* You must be using {ocp} 4.10 or later.
+* You must be using {ocp} 4.10 or later. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
Comment (msugakov) [inline modules/install-acs-operator.adoc:13]:
```suggestion
* You must be using {ocp} {rhacs-operator-min-ocp-version} or later. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```

File: modules/install-central-operator.adoc [hunk 11]
```diff
@@ -14,7 +14,7 @@ When you install {product-title} for the first time, you must first install the
 ====
 
 .Prerequisites
-* You must be using {ocp} 4.10 or later.
+* You must be using {ocp} 4.10 or later. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
Comment (msugakov) [inline modules/install-central-operator.adoc:17]:
```suggestion
* You must be using {ocp} {rhacs-operator-min-ocp-version} or later. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```

File: modules/install-acs-operator-cloud.adoc (hunk 1 above)
Comment (msugakov) [reply modules/install-acs-operator-cloud.adoc]: Confirming resolved. There are more places to use `{rhacs-operator-min-ocp-version}` though. Highlighted them with comments.

File: modules/rhcos-enable-node-scan.adoc [hunk 12]
```diff
@@ -9,7 +9,7 @@
 If you use {ocp}, you can enable scanning of Red Hat Enterprise Linux CoreOS (RHCOS) nodes for vulnerabilities by using {rh-rhacs-first}.
 
 .Prerequisites
-* You must use {ocp} 4.10 or later for scanning RHCOS node hosts.
+* For scanning RHCOS node hosts, you must have installed both Central and Secured Cluster on {ocp} 4.10 or later. For more details on supported managed and self-managed {ocp} versions, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
Comment (msugakov) [inline modules/rhcos-enable-node-scan.adoc:12]:
```suggestion
* For scanning RHCOS node hosts, you must have installed Secured Cluster on {ocp} 4.10 or later. For more details on supported managed and self-managed {ocp} versions, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```

**Comment by gaurav-nelson** [comment]
@msugakov I've addressed all comments except the version number one. Since this PR applies to all docs from `3.73` onwards.

File: installing/acs-installation-platforms.adoc [hunk 13]
```diff
@@ -6,7 +6,7 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-{rh-rhacs-first} and {rh-rhacscs-first} are supported on {ocp} and Kubernetes platforms.
+{rh-rhacs-first} are supported on {ocp} and Kubernetes platforms. For more details about supported self-managed and managed platforms, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
[the remark this answers is not in the document]
Comment (msugakov) [reply installing/acs-installation-platforms.adoc]: Confirming resolved.

File: installing/acs-installation-platforms.adoc [hunk 14]
```diff
@@ -22,23 +22,23 @@ Not all installation methods are supported for all platforms.
...
+|*Platform type ^[1]^*|*Platform ^[2]^*|*Supported for Central*|*Supported for Secured Clusters*|*Supported installation methods*|*Installation steps*
 
 .6+|Managed service platform
 |{osp} Dedicated (OSD)
-|Yes ^[2]^
-|Yes ^[2]^
-.3+d|Operator (recommended), Helm charts, or roxctl CLI ^[3]^
+|Yes
+|Yes
+.3+d|Operator (recommended), Helm charts, or `roxctl` CLI ^[3]^
```
[the remark this answers is not in the document]
Comment (msugakov) [reply installing/acs-installation-platforms.adoc:31]: Confirming resolved.

File: installing/acs-installation-platforms.adoc [hunk 15]
```diff
@@ -72,7 +72,7 @@ Not all installation methods are supported for all platforms.
 [.small]
 --
 1. The availability of support for each platform depends on the overarching lifecycle of the platform and the end-of-life date.
-2. {rh-rhacs-first} is supported as a self-managed add-on for managed OpenShift services that meet the installation and sizing requirements for {product-title-short}. Deploying and managing {product-title-short} components does not fall under the service description for the managed service, and therefore {product-title-short} is not included in the SRE service.
+2. For more details about supported self-managed and managed platforms, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
 3. Do not use the `roxctl` installation method unless you have a specific installation that requires this method.
```
[the remark this answers is not in the document]
Comment (msugakov) [reply installing/acs-installation-platforms.adoc]: Confirming resolved.

File: modules/rhcos-enable-node-scan.adoc [hunk 16]
```diff
@@ -9,7 +9,7 @@
 If you use {ocp}, you can enable scanning of Red Hat Enterprise Linux CoreOS (RHCOS) nodes for vulnerabilities by using {rh-rhacs-first}.
 
 .Prerequisites
-* You must use {ocp} 4.10 or later for scanning RHCOS node hosts.
+* For scanning RHCOS node hosts of the Secured cluster,, you must have installed both Central and Secured cluster on {ocp} 4.10 or later. For more details on supported managed and self-managed {ocp} versions, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
[the remark this answers is not in the document]
Comment (msugakov) [reply modules/rhcos-enable-node-scan.adoc]: Confirming resolved.

File: modules/acs-requirements.adoc [hunk 17]
```diff
@@ -25,11 +29,17 @@ For example, Intel processors older than _Sandy Bridge_ and AMD processors older
 To install {product-title}, you must have:
 
 ifdef::openshift[]
-* {ocp} version {ocp-supported-version} or later
+* {ocp} version {ocp-supported-version} or later. For more details about supported self-managed and managed {ocp} versions, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
+* Cluster nodes with a supported operating system:
+** Red Hat Enterprise Linux CoreOS (RHCOS), Red Hat Enterprise Linux (RHEL) 8.4, or RHEL 8.5.
```
[the remark this answers is not in the document]
Comment (msugakov) [reply modules/acs-requirements.adoc]: Looks good without versions. Confirming resolved.

File: modules/acs-requirements.adoc [hunk 18]
```diff
@@ -25,11 +29,17 @@ For example, Intel processors older than _Sandy Bridge_ and AMD processors older
...
 endif::openshift[]
-* Cluster nodes with a supported operating system.
-For more information, see the link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
+
+ifdef::other-platforms[]
```
[the remark this answers is not in the document]
Comment (msugakov) [reply modules/acs-requirements.adoc:45]: Confirming resolved.

**Review by msugakov — APPROVED** [review]
/lgtm

File: installing/acs-installation-platforms.adoc [hunk 19]
```diff
@@ -6,7 +6,7 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-{rh-rhacs-first} and {rh-rhacscs-first} are supported on {ocp} and Kubernetes platforms.
+{rh-rhacs-first} is supported on {ocp} and Kubernetes platforms. For more details about supported self-managed and managed platforms, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
Comment (mjpytlak) [inline installing/acs-installation-platforms.adoc:9]:
```suggestion
{rh-rhacs-first} is supported on {ocp} and Kubernetes platforms. For more information about supported self-managed and managed platforms, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
I believe we standardize on `information` [1]
[1] https://redhat-documentation.github.io/supplementary-style-guide/#cross-references

File: modules/acs-requirements.adoc [hunk 20]
```diff
@@ -25,11 +37,17 @@ For example, Intel processors older than _Sandy Bridge_ and AMD processors older
 To install {product-title}, you must have:
 
 ifdef::openshift[]
-* {ocp} version {ocp-supported-version} or later
+* {ocp} version {ocp-supported-version} or later. For more details about supported self-managed and managed {ocp} versions, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
Comment (mjpytlak) [inline modules/acs-requirements.adoc:40]: General comment for the PR. Please replace "For more details" with "For more information".

File: modules/rhcos-enable-node-scan.adoc [hunk 21]
```diff
@@ -9,7 +9,7 @@
 If you use {ocp}, you can enable scanning of Red Hat Enterprise Linux CoreOS (RHCOS) nodes for vulnerabilities by using {rh-rhacs-first}.
 
 .Prerequisites
-* You must use {ocp} 4.10 or later for scanning RHCOS node hosts.
+* For scanning RHCOS node hosts of the Secured cluster, you must have installed Secured cluster on {ocp} 4.10 or later. For more details on supported managed and self-managed {ocp} versions, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
Comment (mjpytlak) [inline modules/rhcos-enable-node-scan.adoc:12]: I do not have access to your common attributes file. Should the first instance of Red Hat Enterprise Linux CoreOS an subsequent instances of RHCOS be variables?

**Review by mjpytlak — COMMENTED** [review]
@gaurav-nelson Only a few small comments. However, please provide the links to your updated topics. In particular, I would like to verify the updates in tables render correctly.

**Comment by gaurav-nelson** [comment]
Thank you @mjpytlak I've updated the comment to include previews for the modified items. Table preview is at https://60483--docspreview.netlify.app/openshift-acs/latest/installing/acs-installation-platforms.html#install-platforms-methods_acs-installation-platforms

**Comment by gaurav-nelson** [comment]
Thanks mate! Fixed those in the recent push, and a few more by find - replace.

File: modules/acs-requirements.adoc [hunk 22]
```diff
@@ -5,10 +5,22 @@
 [id="acs-general-requirements_{context}"]
 = General requirements
 
-ifeval::["{context}" == "install-rhacs-ocp"]
+ifeval::["{context}" == "prerequisites-ocp"]
 :openshift:
 endif::[]
 
+ifeval::["{context}" == "prerequisites-other"]
```
Comment (mjpytlak) [inline modules/acs-requirements.adoc:12]: @gaurav-nelson Sorry missed this the first time through, but you do not close these ifeval directives at the end of the file.

**Pull request merged**